docs/source/user-guide/flightsql.md

```diff
@@ -103,6 +92,9 @@ The important pieces of information:
 | Advanced Options | useEncryption=false                                |
 | URL              | jdbc:arrow-flight://127.0.0.1:50050                |
 
+**Note**: The `useEnryption` flag will not be required if [this PR](https://github.com/spaceandtimelabs/arrow-ballista/pull/15) is merged,
```

Comment:
Is there a reason we can't have this PR open against the arrow-ballista repo?

Reply:
I think it should be @andygrove . Probably @iajoiner opened it before the FlightSQL changes from our fork were upstreamed?

Reply:
I will close that one and open a new one.